internals/overlord/logstate/gatherer.go

```diff
@@ -311,7 +312,8 @@ type logClient interface {
 
 func newLogClient(target *plan.LogTarget) (logClient, error) {
 	switch target.Type {
-	//case plan.LokiTarget: TODO
+	case plan.LokiTarget:
```

Comment:
Per recent lesson learned about `UnsetLevel` (level of what?), maybe this should be `LokiLogTarget`.

Reply:
So the constants we've defined here are
https://github.com/canonical/pebble/blob/5d21565361718f3311bd181649e34957436445e9/internals/plan/plan.go#L502-L504

We've preemptively dodged the `UnsetLevel` problem by defining `UnsetLogTarget`. For the other two, both "Loki" and "syslog" refer unambiguously to logging, so I think throwing the word "Log" in here is probably unnecessary. (And `SyslogLogTarget` would be a bit ridiculous, haha).